gslib/utils/text_util.py

```diff
@@ -245,9 +272,110 @@ def PrintableStr(input_val):
   Returns:
     (str) A UTF-8 encoded string, or None.
   """
-  return input_val.encode(UTF8) if input_val is not None else None
+  return input_val
```

Comment:
This... no longer does what the docstring says. In fact, it basically does nothing now. Do we still need this method?

Reply:
I see no reason to keep it... there is really only one block that uses it in the same file around lines 341 - 343. My vote is to take it out.

Reply:
Removed method and calls. Tests that use that section of code have been tested and are still working.